script/configs/custom_analysis.yaml

```diff
@@ -8,6 +8,11 @@
 # from a top-level package into more specific packages in order to incrementally
 # migrate a federated plugin.
 #
+# DO NOT move or delete this file without updating
+# https://github.com/dart-lang/sdk/blob/master/tools/bots/flutter/analyze_flutter_plugins.sh
+# which references this file from source, but out-of-repo.
+# Contact stuartmorgan or devoncarew for assistance if necessary.
+
```

Comment:
Wouldn't it make more sense to have this warning in the cirrus file next to the invocation there?

Reply:
Good thought, that's more likely to be noticed in review. I left this longer explanation here, and added a shorter warning to .cirrus.yml that references it.